skycatalogs/objects/gaia_object.py

```diff
@@ -372,22 +371,19 @@ def mjd(self):
         return self._mjd
 
     def __getitem__(self, key):
+        cols = ('id', 'ra_deg', 'dec_deg', 'phot_bp_mean_flux',
+                'phot_rp_mean_flux')
         if isinstance(key, int) or isinstance(key, np.int64):
-            row = {col: self.df[col][key] for col in ('id', 'ra_deg',
-                                                      'dec_deg',
-                                                      'phot_bp_mean_flux',
-                                                      'phot_rp_mean_flux')}
+            row = {col: self.df[col][key] for col in cols}
             return GaiaObject(row, self, key)
 
         elif type(key) == slice:
```

Comment:
Any reason not to use `isinstance(key, slice)` here?

Reply:
No reason.  I'll change it.